lib/send_text.rb

```diff
@@ -0,0 +1,18 @@
+require 'twilio-ruby'
+
+class SendText
+  def send_text
+    time = Time.new
+    account_sid = "ACad78197ffa4e768e01d488004100210e"
```

Comment:
I can show you how to use dotenv to store these. You should remove these from your code and push again.